packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -284,22 +285,19 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 					writeDeployConfig(resolvedPluginConfig, resolvedViteConfig);
 				}
 			},
-			handleHotUpdate(options) {
+			hotUpdate(options) {
```

Comment:
Changed from `handleHotUpdate` to `hotUpdate` so it runs for each environment. Otherwise `[]` is only returned for the default `client` and `ssr` environments.

Reply:
TIL!

Reply:
same 😄